### PR TITLE
Generated model and controller for Merchant class. Added Route resour…

### PR DESCRIPTION
**Pull request by mdove92**
Generated model and controller for Merchant class. Added Route resources for Merchant

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :works
```
Comment (tofuandeve) [inline config/routes.rb]:
```suggestion
  resources :merchants
```

**Review by tofuandeve — CHANGES_REQUESTED** [review]

**Review by tofuandeve — APPROVED** [review]
Good job!

**Review by michaela260 — APPROVED** [review]
YAYYYY 1ST PULL REQUEST WOOOHOOOO!!! #bandits

File: app/helpers/merchants_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module MerchantsHelper
+end
```
Comment (emaust) [inline app/helpers/merchants_helper.rb:2]: NOICE

**Review by emaust — APPROVED** [review]
Noice noice noice

**Pull request merged**